### DISCUSSION TITLE
Share links do not open in a new tab

### DISCUSSION THREAD

**Discussion by dontcrash** [jpanther/congo#562] [Ideas]
When enabling article.sharingLinks, once clicked they open in the same tab, a better scenario is to have them open in a new tab thus keeping the reader on your site

This can be achieved with adding the following html attributes to the a element:

```html
target="_blank" rel="noreferrer noopener"
```

**Comment by jpanther** [comment]
Thanks for the suggestion. This behaviour will be changed in the next release.
